user-manager-service/src/main/java/com/crm/food/establishment/user/validation/ValidationRegexps.java

```diff
@@ -0,0 +1,10 @@
+package com.crm.food.establishment.user.validation;
+
+public class ValidationRegexps {
```

Comment:
Actually not:
<img width="455" alt="image" src="https://github.com/polinaucc/crm-food-establishment/assets/95944057/80dbd547-7c78-419f-a031-5c93b0a09261">
<img width="288" alt="image" src="https://github.com/polinaucc/crm-food-establishment/assets/95944057/7609f66c-5162-4cb9-9714-ce4c6f20a272">
And I thought It would be good practice to keeps them in one place